### PR TITLE
Add JHipster-lite docker release and instruction to use JHipster lite through docker

### PR DESCRIPTION
**Pull request by PierreBesson**
It can be tried out using: `pbesson/jhlite:latest`

### CODE REVIEW

File: .github/workflows/docker-image-publish.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  push:
+    branches:
+      - main
+    # Publish semver tags as releases.
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```
Comment (pascalgrimaud) [inline .github/workflows/docker-image-publish.yml:33]: maybe 22 instead of 20 ?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,22 @@ JHipster is a development platform to quickly generate, develop & deploy modern
 - you will only generate the code you want, no additional unused code
 - best quality as possible: 💯% coverage, 0 code smell, no duplication 😎
 
+## Docker Quickstart
+
+To start a local instance of JHipster lite using docker, go to your desired application folder and run:
```
Comment (qmonmert) [inline README.md:31]:
``` diff
+ Lite
- lite
```

File: .github/workflows/docker-image-publish.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+      - name: Login to Docker Registry
+        env:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+        if: ${{env.DOCKERHUB_USERNAME != '' && startsWith(github.ref, 'refs/tags/v')}}
+        uses: docker/login-action@v2.0.0
+        id: login-hub
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```
Comment (PierreBesson) [inline .github/workflows/docker-image-publish.yml:66]: @pascalgrimaud this will need some secrets to be provisionned.

**Comment by pascalgrimaud** [comment]
Thanks for your work @PierreBesson 
Let's merge it

File: .github/workflows/docker-image-publish.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v4.0.1
+        with:
+          images: ghcr.io/${{ github.repository }}${{ (steps.login-hub.outcome == 'success' && ',jhipster/jhipster-lite) || ''}}
```
Comment (pascalgrimaud) [inline .github/workflows/docker-image-publish.yml:72]: @PierreBesson : this line doesn't work

**Pull request merged**